.readthedocs.yaml

```diff
@@ -0,0 +1,29 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "3.8"
+    # You can also specify other tool versions:
+    # nodejs: "16"
+    # rust: "1.55"
+    # golang: "1.17"
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+   configuration: docs/conf.py
+
+# If using Sphinx, optionally build your docs in additional formats such as PDF
```

Comment:
That is nice to be able to do that for offline work

Reply:
`.readthedocs.yaml` is specifically to configure online builds on readthedocs. For offline work, you can always build PDF documentation offline with `make latexpdf` inside the `docs` folder, or with `make -C docs latexpdf`.

Did you want pdf automatic builds, or is the make target fine?

Reply:
Make target is fine, I hadn't consider this was the readthedocs.yaml